errors/go.mod

```diff
@@ -5,18 +5,13 @@ go 1.20
 require (
 	github.com/pkg/errors v0.9.1
```

Comment:
I would also propose removing the errors pkg here as it's only needed for stack traces. Then we would make stack tracers via a build flag and only at the core layer. There isn't a need for all errors to have stack traces here